main/src/main/res/values/strings.xml

```diff
@@ -336,6 +336,7 @@
     <string name="vpn_disallow_radio">VPN is used for all apps but exclude selected</string>
     <string name="vpn_allow_radio">VPN is used for only for selected apps</string>
     <string name="vpn_allow_bypass">Allow apps to bypass the VPN</string>
+    <string name="enable_allow_mobile">Enable extra setting for mobile data</string>
```

Comment:
I don't think  "extra setting for mobile data" is a good wording. It is heard to understand what that does.